meta/implementation.md

```diff
@@ -141,3 +141,17 @@ To make `date_shift` easier to implement, we have found some libraries that foll
 - JavaScript: [Moment.js](https://momentjs.com/)
 - Python: [dateutil](https://dateutil.readthedocs.io/en/stable/index.html)
 - R: [lubridate](https://lubridate.tidyverse.org/) ([Cheatsheet](https://rawgit.com/rstudio/cheatsheets/master/lubridate.pdf))
+
+## Quantile algorithms
+
+The `quantiles` could implement a number of different algorithms, literature usually distinguishes [9 types](https://en.wikipedia.org/wiki/Quantile#Estimating_quantiles_from_a_sample).
+Right now it's not possible to choose from them, but it may be added in the future.
+To improve interoperability openEO processes v1.2 added details about the algorithm that should be implemented.
+A survey has shown that most libraries implement type 7 and as such this was chosen to be the default.
+
+We have found some libraries that can be used for an implementation:
+- Java: [Apache Commons Math Percentile](http://commons.apache.org/proper/commons-math/javadocs/api-3.6/org/apache/commons/math3/stat/descriptive/rank/Percentile.html), choose the [estimation type `R_7`](http://commons.apache.org/proper/commons-math/javadocs/api-3.6/org/apache/commons/math3/stat/descriptive/rank/Percentile.EstimationType.html#R_7)
+- JavaScript: [d3](https://github.com/d3/d3-array/blob/v2.8.0/README.md#quantile), implements type 7 only.
```

Comment:
Change wording for better understanding --> has only type 7 implemented

Reply:
Thanks, done.